Dockerfile

```diff
@@ -18,8 +19,9 @@ COPY . ${GOPATH}/src/code.gitea.io/gitea
 WORKDIR ${GOPATH}/src/code.gitea.io/gitea
 
 #Checkout version if set
-RUN if [ -n "${GITEA_VERSION}" ]; then git checkout "${GITEA_VERSION}"; fi \
- && make clean-all build
+RUN if [ -n "${GITEA_VERSION}" ]; then git checkout "${GITEA_VERSION}"; fi && \
+ export CGO_CFLAGS="$(go env CGO_CFLAGS) -DSQLITE_MAX_VARIABLE_NUMBER=32766 $CGO_EXTRA_CFLAGS" && \
```

Comment:
Isn't this $CGO_EXTRA_CFLAGS superfluous with the variable before?

Reply:
Indeed, let me revert the changes to dockerfile.

Reply:
I've pushed changes, and will review CI output before merge.